lib/services/rating_service.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:project_eureka_flutter/models/rating_model.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart' as DotEnv;
+import 'dart:async';
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+class RatingService {
+  Future<RatingModel> updateRating(RatingModel rating) async {
```

Comment:
I believe this function can be changed to `Future<void>` as I don't think you return anything to the rating page.